pipeline_test.py

```diff
@@ -0,0 +1,44 @@
+import os
+import subprocess
+from subprocess import Popen, PIPE
+
+import pydicom
+
+if __name__ == '__main__':
+    cmd_1 = "./build/exe/qctool --config-file='./schema/useCase.json'"
+    cmd_2 = "storescp 11113 --output-directory ./result/"
+    cmd_3 = "storescp 11114 --output-directory ./result_quarantine/"
+    cmd_4 = "storescu localhost 11112 DICOM_Images --scan-directories"
+    cmd = cmd_1 + " & " + cmd_2 + " & " + cmd_3 + " & " + cmd_4
+    os.system(cmd)
+
+    '''
+    Note: when we rerun the whole pipeline test,
```

Comment:
can't you fill the processes at the end of the test?

Reply:
I will figure out a way to shut down the pipeline, DCMTK sender & receiver using python in the future.